apparmor.d/groups/kde/kded5

```diff
@@ -29,16 +30,59 @@ profile kded5 @{exec_path} {
 
   signal (send) set=hup peer=xsettingsd,
 
+  dbus receive bus=system path=/org/freedesktop/NetworkManager/SecretAgent
+       interface=org.freedesktop.NetworkManager.SecretAgent
+       member=CancelGetSecrets
+       peer=(label=NetworkManager),
+
+  dbus receive bus=system path=/org/freedesktop/NetworkManager/SecretAgent
+       interface=org.freedesktop.NetworkManager.SecretAgent
+       member=CancelGetSecrets
+       peer=(label=NetworkManager),
+
+  dbus receive bus=system path=/org/freedesktop/NetworkManager/AccessPoint/@{int}
+       interface=org.freedesktop.DBus.Properties
+       member=PropertiesChanged
+       peer=(label=NetworkManager),
+
+  dbus receive bus=system path=/org/freedesktop/NetworkManager/Devices/@{int}
+       interface=org.freedesktop.DBus.Properties
+       member={PropertiesChanged,AccessPointAdded,AccessPointRemoved}
+       peer=(label=NetworkManager),
+
+  dbus receive bus=system path=/org/freedesktop/PolicyKit1{,/**}
```

Comment:
Try to use `abstractions/bus/org.freedesktop.PolicyKit1` instead, only add rule here if it requires more (or even give full talk access: https://apparmor.pujol.io/development/dbus/#dbus-directive